tests/core/util/test_keyring_wrapper.py

```diff
@@ -482,3 +484,97 @@ def test_passphrase_hint_update(self):
             KeyringWrapper.get_shared_instance().get_master_passphrase_hint()
             == "Something you wouldn't expect The Shredder to say"
         )
+
```

Comment:
I would add test cases for labels containing Emoji, Japanese text, and an RTL language like Arabic/Hebrew.

Reply:
See f126660655451c42c06ece4994a9cd09e2ec385a